Components/Options/RooDropdown.cs

```diff
@@ -36,7 +36,7 @@ public class RooDropdown : Selectable, IPointerClickHandler, ISubmitHandler, ICa
 
         public DropDownEvent OnValueChanged { get; set; } = new();
 
-        protected override void Awake()
+        public override void Awake()
```

Comment:
is there a reason why these event functions were change to be public?

Reply:
The nuget is publicizied the project would complain about wrong modifier when overriding.

Reply:
ahh didn't know that! thanks.